.github/templates/windows_ci_workflow.yml.in

```diff
@@ -44,9 +44,20 @@ jobs:
         shell: bash
         run: |
           .jenkins/pytorch/win-build.sh
-      - name: Upload artifacts
+      # Upload to github so that people can click and download artifacts
+      - name: Upload artifacts to Github
         if: always()
         uses: actions/upload-artifact@v2
+        # Don't fail on upload to GH since it's only for user convenience
+        continue-on-error: true
+        with:
+          retention-days: 30
```

Comment:
Same as above, two weeks should be enough, shouldn't it?

Reply:
Done